<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtFilterOperationerMainDialog</class>
 <widget class="QDialog" name="QtFilterOperationerMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilterOperationer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_original">
      <attribute name="title">
       <string>&amp;1. Original</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="button_load">
         <property name="text">
          <string>&amp;Load original</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="area_source">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="content_source">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>63</width>
            <height>16</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_filter">
      <attribute name="title">
       <string>&amp;2. Filter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QWidget" name="widget_filter_rows" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_filter_cols">
            <property name="text">
             <string>Columns</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="box_filter_cols">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1024</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer_filter_rows">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_filter_cols" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_filter_rows">
            <property name="text">
             <string>Rows</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="box_filter_rows">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1024</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer_filter_cols">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableView" name="filter"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_result">
      <attribute name="title">
       <string>&amp;3. Result</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="button_do">
            <property name="text">
             <string>&amp;Generate result</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_save">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="area_result">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="content_result">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>63</width>
            <height>16</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
